goss-testing/automated/run-ncn-tests.sh

```diff
@@ -659,6 +659,7 @@ function add_local_vars {
     var_string+="\n${is_vshasta}\n"
 
     echo -e "${var_string}" >> "$1"
+    # shellcheck disable=SC2320
```

Comment:
This is a valid issue. Why are we ignoring it?

Reply:
My understanding of this code is that we indeed using return code of `echo` command as return code of this function. For example, if `"$1"` is a read-only file, it will be non-zero.

Reply:
Ok, that makes sense. thanks.